### DISCUSSION TITLE
ESPWiFiAnalyzer adapted to TFT_e_SPI gives me an error

### DISCUSSION THREAD

**Discussion by TangerineDreamer** [moononournation/Arduino_GFX#309] [Q&A]
Hi moononourstation,
 i'm trying to adapt the sketch ESPWiFiAnalyzer to TFT_e_SPI,but this error occurs:

error: no matching function for call to 'WiFiClass::BSSID(int&, uint8_t [6])'
       WiFi.BSSID(i, bssidA);

does it come from WiFi.h or your library?
how to resolve that?

**Comment by TangerineDreamer** [comment]
I did adapt the picowifianalyzer and took those variables:
  uint8_t bssidA[6];
  uint8_t bssidB[6];
that gave me errors,so i took the variables from espwifianalyzer:  uint8_t *bssid; and it was okay,
by the way it's the espwifianalyzer sketch, that fits better :)

**Comment by moononournation** [comment]
This is the discussion area of Arduino_GFX, please ask your question in the right place.
